cli/build.gradle.kts

```
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 SpecMesh Contributors (https://github.com/specmesh)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    application
+}
+
+val kafkaVersion : String by extra
+val spotBugsVersion : String by extra
+val jacksonVersion : String by extra
+val lombokVersion : String by extra
+val confluentVersion : String by extra
+val testcontainersVersion : String by extra
+
+
+dependencies {
+    implementation("info.picocli:picocli:4.7.1")
+
+    implementation(project(":parser"))
+    implementation(project(":kafka"))
+    testImplementation(project(":kafka-test"))
+
+    implementation("org.testcontainers:testcontainers:$testcontainersVersion")
+    implementation("org.testcontainers:kafka:$testcontainersVersion")
```

I'd kind of hope these should be:

```suggestion
    testImplementation("org.testcontainers:testcontainers:$testcontainersVersion")
    testImplementation("org.testcontainers:kafka:$testcontainersVersion")
```

???